warehouse/models/gtfs_schedule_latest_only/agency.md

```diff
@@ -0,0 +1,33 @@
+Original definitions from https://gtfs.org/reference/static#agencytxt
```

Comment:
Can `gtfs_views` access docs MD files in this directory? A lot of these fields are shared across the corresponding `dim` table, just wondering if we want to move these markdown files somewhere else where it's accessible in both places

Reply:
all dbt artifacts are global, including docs, so we can pick whatever location we want